src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -529,25 +533,24 @@ method getNumberOfPinnedMessages*(self: Module): int =
 method updateContactDetails*(self: Module, contactId: string) =
   let updatedContact = self.controller.getContactDetails(contactId)
   for item in self.view.model().modelContactUpdateIterator(contactId):
-    if(item.senderId == contactId):
+    if item.senderId == contactId:
       item.senderDisplayName = updatedContact.defaultDisplayName
       item.senderOptionalName = updatedContact.optionalName
       item.senderIcon = updatedContact.icon
       item.senderColorHash = updatedContact.colorHash
       item.senderIsAdded = updatedContact.details.added
       item.senderTrustStatus = updatedContact.details.trustStatus
       item.senderEnsVerified = updatedContact.details.ensVerified
-    if(item.quotedMessageAuthorDetails.details.id == contactId):
+
+    if item.quotedMessageAuthorDetails.details.id == contactId:
       item.quotedMessageAuthorDetails = updatedContact
       item.quotedMessageAuthorDisplayName = updatedContact.defaultDisplayName
       item.quotedMessageAuthorAvatar = updatedContact.icon
-    if(item.messageContainsMentions):
-      let (message, _, err) = self.controller.getMessageDetails(item.id)
-      if(err.len == 0):
-        let chatDetails = self.controller.getChatDetails()
-        let communityChats = self.controller.getCommunityById(chatDetails.communityId).chats
-        item.messageText = self.controller.getRenderedText(message.parsedText, communityChats)
-        item.messageContainsMentions = message.containsContactMentions()
```

Comment:
I removed the check/update of `messageContainsMentions`, because I don't a world where updating the contact details would make a message no longer a mention. Let me know if I'm missing something